### ISSUE TITLE
include failed errors in Nextcloud log, no sensordata created

### ISSUE THREAD

**Issue by rohklich** [alexstocker/sensorlogger#83]
**Describe the bug**
Tried to create a simple log to test the app. Getting HTTP 405(Method Not Allowed) **everytime**.
After logging in to Nextcloud and taking a look to the error log I see these 2 errormessages alternating(triggered on every reload):

`include(sensorlogger/admin.php): failed to open stream: No such file or directory at /html/nextcloud/lib/private/legacy/app.php#623`

`include(): Failed opening 'sensorlogger/admin.php' for inclusion (include_path='/html/nextcloud/3rdparty/pear/archive_tar:/html/nextcloud/3rdparty/pear/console_getopt:/html/nextcloud/3rdparty/pear/pear-core-minimal/src:/html/nextcloud/3rdparty/pear/pear_exception:/html/nextcloud/apps') at /html/nextcloud/lib/private/legacy/app.php#623`

I uninstalled and then reinstalled the app, but only to see the 2 errors from above again and again. The errors happened without any request after the reinstall.
Maybe there should be some settings invoked for sensorlogger, but NC couldn't find the file and produces errors as long as I stay on the errorlog page(in NC settings)...

**To Reproduce**
1. Install sensorlogger
2. Go to errorlog in settings
3. Refresh page

**Version:**
Nextcloud 16.0.3
sensorlogger 0.1.1 (installed via 'Apps' in Nextcloud)

**Comment by alexstocker** [comment]
Hi @rohklich 
Thanx for reporting this issue. I will try to reproduce your issue asap.

**Comment by alexstocker** [comment]
How did you POST your request. 405 indicates that you'r trying to execute a GET request to the [API Endpoint](https://github.com/alexstocker/sensorlogger/wiki/API#simple). Try to use https://github.com/alexstocker/sensorlogger/blob/0.1.1nc/tests/curl/post.php as example if you are used to PHP and do not forget to edit $url, $username and $token vars to fit to your needs.

**Comment by rohklich** [comment]
I used a chrome extension. Used POST, with following headers:
`Authorization: Basic ......
Content-Type: application/json`
and payload from your wiki:
`{
  "date":"2017-03-03 12:30:47",
  "deviceId":"f7645058-fe8c-11e6-bc64-92361f002671",
  "humidity":34.700000762939,
  "temperature":24
}`

**Comment by rohklich** [comment]
Ok, now I managed to push sensordata successful to NC. I have no idea, what I did different now. But still I have those errors in the log. I don't know if I this will be a problem in the future, for now it seems to work.

I just saw, that you register a settings page there:
https://github.com/alexstocker/sensorlogger/blob/38c86d1cd667910022b9dcc3169d3e64fcc042c7/appinfo/app.php#L30
but I couldn't find the corresponding admin.php (NC obviously couldn't find it either)
